### PR TITLE
Improvement: adjust timeout for builds with remote access

### PR DESCRIPTION
**Pull request by artemii-yanushevskyi**

### CODE REVIEW

File: content/troubleshooting/accessing-builder-machine-via-ssh.md [hunk 1]
```diff
@@ -58,10 +58,10 @@ You can use the remote session to reproduce all commands run during the build, r
 
 ## Remote access session time limit
 
-The SSH or VNC/RDP session remains active for a maximum of 20 minutes after all build steps are completed or until the maximum build duration limit is reached, whichever comes first. If you log out from the virtual machine during an active VNC or SSH/RDP session, the build is finished automatically, and you cannot access the builder for this session. You would have to run a new build to establish a new remote access connection.
+Remote access is available during and 10 minutes after the build finishes. Once connected, the SSH or VNC/RDP session remains active until the maximum build duration limit is reached. Even if the session ends sooner, you can still connect to the virtual machine.
```
[the remark this answers is not in the document]
Comment (helinanever) [reply content/troubleshooting/accessing-builder-machine-via-ssh.md:61]: I think it's not clear enough. What are the ways to end the session?
- end the remote access session - does this terminate the build?
- cancel the build
- build timeout

File: content/troubleshooting/accessing-builder-machine-via-ssh.md [hunk 2]
```diff
@@ -58,10 +58,10 @@ You can use the remote session to reproduce all commands run during the build, r
...
 
 {{<notebox>}}
-**Note:** If you have finished debugging the build, don't forget to end the session to avoid losing build time.
+**Note:** If you no longer need to debug the build, don't forget to cancel the build to avoid losing build time.
```
[the remark this answers is not in the document]
Comment (helinanever) [reply content/troubleshooting/accessing-builder-machine-via-ssh.md:64]: Here too I have a question: does one need to cancel the build to end the session?

**Review by helinanever — CHANGES_REQUESTED** [review]

File: content/troubleshooting/accessing-builder-machine-via-ssh.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mikhail-tokarev) [reply content/troubleshooting/accessing-builder-machine-via-ssh.md:61]: I think original text is better
> The build virtual machine (VM) will remain available for an SSH connection for 10 minutes after the build finishes running and then automatically shut down (or you can cancel it).

[the remark this answers is not in the document]
Comment (mikhail-tokarev) [reply content/troubleshooting/accessing-builder-machine-via-ssh.md:61]: also
> After you SSH into the build, the connection will remain open until the maximum build duration limit is reached (60 minutes by default).

File: content/troubleshooting/accessing-builder-machine-via-ssh.md [hunk 3]
```diff
@@ -58,12 +58,7 @@ You can use the remote session to reproduce all commands run during the build, r
 
 ## Remote access session time limit
 
-The SSH or VNC/RDP session remains active for a maximum of 20 minutes after all build steps are completed or until the maximum build duration limit is reached, whichever comes first. If you log out from the virtual machine during an active VNC or SSH/RDP session, the build is finished automatically, and you cannot access the builder for this session. You would have to run a new build to establish a new remote access connection.
-
-{{<notebox>}}
-**Note:** If you have finished debugging the build, don't forget to end the session to avoid losing build time.
-{{</notebox>}}
-
+The build virtual machine will remain available for an SSH or VNC/RDP connection for 10 minutes after the build finishes running. Once connected, the virtual machine remains available until build is cancelled or the maximum build duration limit is reached (60 minutes by default).
```
Comment (helinanever) [inline content/troubleshooting/accessing-builder-machine-via-ssh.md:61]:
```suggestion
The virtual machine running the build will remain available for an SSH or VNC/RDP connection for 10 minutes after the build finishes running. Once connected, the session remains active until the build is canceled or the maximum build duration limit is reached (60 minutes by default).
```

Comment (artemii-yanushevskyi) [reply content/troubleshooting/accessing-builder-machine-via-ssh.md:61]: I thought to be more general saying "virtual machine remains available" to emphasize that it's possible to reconnect

Maybe "Once connected, the debug session remains active until [...]"

Comment (helinanever) [reply content/troubleshooting/accessing-builder-machine-via-ssh.md:61]: Yep, this sounds good to me.

**Review by helinanever — APPROVED** [review]
LGTM

**Pull request merged**